tools/tsp-client/test/commands.spec.ts

```diff
@@ -48,6 +53,16 @@ describe.sequential("Verify commands", () => {
     }
   });
 
+  it("Generate lock file with altername package path", async () => {
+    try {
+      await generateLockFileCommand({"emitter-package-json-path": joinPaths(repoRoot, "tools/tsp-client/test/utils/alternate-emitter-package.json")});
+
+      assert.isTrue((await stat(joinPaths(repoRoot, "tools/tsp-client/test/utils/alternate-emitter-package.json"))).isFile());
```

Comment:
Should we delete the lock file you have checked in under utils at the beginning of this test, then in this assert we should check for its existence?